tests/e2e/core-tests/specs/shopper/front-end-checkout-create-account.test.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests, jest/expect-expect */
+/**
+ * Internal dependencies
+ */
+ const {
+	shopper,
+	merchant,
+	createSimpleProduct,
+	uiUnblocked,
+	setCheckbox,
+    settingsPageSaveChanges,
+} = require( '@woocommerce/e2e-utils' );
+
+/**
+ * External dependencies
+ */
+const {
+	it,
+	describe,
+	beforeAll,
+} = require( '@jest/globals' );
+
+const config = require( 'config' );
+const simpleProductName = config.get( 'products.simple.name' );
+
+const runCheckoutCreateAccountTest = () => {
+	describe('Shopper Checkout Create Account', () => {
+		beforeAll(async () => {
+			await merchant.login();
+			await createSimpleProduct();
+            await merchant.openSettings('account');
```

Comment:
Looks like some extra spacing got added in here on lines 31-33 and line 11 above.

Reply:
Oh, I'm struggling with that all the time, It's my external keyboard problem probably. But realized that pressing CMD+A shows invalid spaces, then I need to copy/paste valid Tab spaces and overwrite them :D

Reply:
Haha, I know the feeling! I noticed my PhpStorm settings for the longest time were doing tabs vs spaces and it kept causing weird formatting issues since it "helped" by making everything look aligned 😓 Anyway thanks for making these changes!